wrappers/services.go

```diff
@@ -404,29 +457,16 @@ func AddSnapServices(s *snap.Info, disabledSvcs []string, opts *AddSnapServicesO
 			// boot
 			continue
 		}
+		if strutil.ListContains(disabledSvcs, app.Name) || preseeding {
```

Comment:
I know this is existing code that was moved around but it got me thinking. Probably not a problem in practice but this may become quadratic. Perhaps worth adding a XXX comment. We could pre-sort `disabledSvcs` and then offer an optimized version with better complexity guarantees.

Reply:
Yes, that's right, although as you say, not a problem in practice. I can add a comment but tbh we have more places like this.